docs/ref/contrib/auth.txt

```diff
@@ -740,6 +740,12 @@ The following backends are available in :mod:`django.contrib.auth.backends`:
         if it wasn't provided to :func:`~django.contrib.auth.authenticate`
         (which passes it on to the backend).
 
+.. class:: AllowAllUsersRemoteUserBackend
+
+    Same as :class:`RemoteUserBackend` except that it doesn't reject inactive
+    users because :attr:`~AllowAllUsersRemoteUserBackend.user_can_authenticate` always
+    returns ``True``.
+
     .. method:: user_can_authenticate()
```

Comment:
I moved this method from the docs for `RemoteUserBackend` to `AllowAllUsersRemoteUserBackend` as that's where it's [uniquely defined](https://github.com/django/django/blob/1e7327770c908cb51f888c9269322247fa3525a5/django/contrib/auth/backends.py#L341) (`RemoteUserBackend` just [inherits](https://github.com/django/django/blob/1e7327770c908cb51f888c9269322247fa3525a5/django/contrib/auth/backends.py#L250) from `ModelBackend`).